pages/product.js

```diff
@@ -0,0 +1,17 @@
+const { I } = inject();
+
+module.exports = {
+  productPrice: { css: '#our_price_display' },
+  addToCart: { css: '#add_to_cart' },
+  proceedToCheckout: { xpath: '//a[@title="Proceed to checkout"]' },
+
+  async getProductPrice() {
+    return await I.grabTextFrom(this.productPrice);
+  },
+
+  clickAddToCart(){
+    I.click(this.addToCart);
+    I.click(this.proceedToCheckout);
```

Comment:
`proceedToCheckout` is not related to `clickAddToCart()`, move to separate method

Reply:
Done